learn/charon/charon-cli-reference.md

```diff
@@ -536,6 +536,57 @@ You can also consider adding [alternative public relays](../../advanced-and-trou
+      --p2p-tcp-address strings                Comma-separated list of listening TCP addresses (ip and port) for libP2P traffic. Empty default doesn't bind to local port therefore only supports outgoing connections.
+      --p2p-udp-address strings                Comma-separated list of listening UDP addresses (ip and port) for libP2P traffic. Empty default doesn't bind to local port therefore only supports outgoing connections.
+      --shutdown-delay duration                Graceful shutdown delay. (default 1s)
+      --src-dir string                         The source charon folder with existing cluster data (lock, validator_keys, etc.). (default ".charon")
+      --timeout duration                       Timeout for the command, should be increased if the command times out. (default 1m0s)
+      --unverified                             When KeyManager is used, charon has no access to the existing validator keys. The flag allows to proceed, but resulting cluster-lock.json becomes unverifiable.
+      --withdrawal-addresses strings           Comma separated list of Ethereum addresses to receive the returned stake and accrued rewards for each validator. Either provide a single withdrawal address or withdrawal addresses for each validator.
+```
+
+Most flags for `charon alpha add-validators` are the same as those for the `charon dkg` command. The most important and required flags are: `--src-dir`, `--dst-dir`, `--num-validators`, `--unverified`, `--withdrawal-addresses`, and `--fee-recipient-addresses`.
+
+There are two typical use cases:
+
+1. If your source cluster keys are present as files in the `validator_keys` subdirectory, you can add new validators using the `--num-validators` flag. The destination folder will contain the updated cluster structure, including all partial keys, deposit data files, and a `cluster-lock.json` file that passes hash and signature verification. The ENR remains unchanged. The new cluster will have a new cluster hash and will appear as a new cluster for observability. Note that the new cluster will not appear in Launchpad, as the `--publish` flag is disabled for this command.
+
+2. If you use a KeyManager and do not have a `validator_keys` subdirectory, the command must use the same KeyManager software where the new keys will be added. In this case, you must specify the `--unverified` flag, which sets empty signatures in the `cluster-lock.json` file. As a result, you will need to use the `--no-verify` flag when running the cluster. This is because the lock signatures are derived from all partial validator signatures, which are not accessible by Charon in this scenario.
+
+After executing this command, you must stop your existing cluster (`--src-dir`) and the corresponding validator clients. Then, start the new cluster (`--dst-dir`) and allow the validator clients to load all keys (old and new). It is acceptable to update and restart all nodes sequentially, one by one.
```

Comment:
Do we want to add this description in the CLI reference? IMO the CLI reference should be simply... that - a reference markdown. I think it would be better to have this under dedicated page in advanced guides? Up to debate though, as it's still in `alpha`. Probably also a bit more detailed step-by-step explanations for the users?